pom.xml

```diff
@@ -49,6 +49,7 @@
 
     <modules>
         <module>dbapi</module>
+        <module>docker</module>
```

Comment:
I think no, because we want to put artefacts to nexus every time. 
We use a special profile for build and push docker images.